api-authorizations/src/test/scala/com/normation/plugins/apiauthorizations/api/UserApiTest.scala

```diff
@@ -0,0 +1,94 @@
+package com.normation.plugins.apiauthorizations
+
+import better.files._
+import com.normation.rudder.AuthorizationType
+import com.normation.rudder.RudderAccount
+import com.normation.rudder.User
+import com.normation.rudder.UserService
+import com.normation.rudder.api.ApiAccount
+import com.normation.rudder.api.ApiAccountId
+import com.normation.rudder.api.ApiAccountKind
+import com.normation.rudder.api.ApiAccountName
+import com.normation.rudder.api.ApiAuthorization
+import com.normation.rudder.api.ApiToken
+import com.normation.rudder.api.ApiVersion
+import com.normation.rudder.rest.RestTestSetUp
+import com.normation.rudder.rest.TraitTestApiFromYamlFiles
+import java.nio.file.Files
+import net.liftweb.common.Loggable
+import org.joda.time.DateTime
+import org.joda.time.DateTimeUtils
+import org.junit.runner.RunWith
+import org.specs2.mutable.Specification
+import org.specs2.runner.JUnitRunner
+import org.specs2.specification.BeforeAfterAll
+
+@RunWith(classOf[JUnitRunner])
+class UserApiTest extends Specification with TraitTestApiFromYamlFiles with Loggable with BeforeAfterAll {
+
+  sequential
+
+  val restTestSetUp = RestTestSetUp.newEnv
+
+  val tmpDir: File = File(Files.createTempDirectory("rudder-test-"))
+  override def yamlSourceDirectory  = "authorizations_api"
+  override def yamlDestTmpDirectory = tmpDir / "templates"
+
+  val fixedDate = DateTime.parse("1970-01-01T00:00:00.000Z")
```

Comment:
Can you use a different, random date in the present ? That specific date and time is too easy to believe as a badly init one ?